src/teamsnap-ui.scss

```diff
@@ -1,3 +1,5 @@
+@import "config/index";
 @import "base/index";
+@import "fonts/index";
```

Comment:
minor nit, probably doesn't matter functionally, but what do you think about moving fonts above config?